std_msgs/Header header

VertexMsg[] vertex
EdgeMsg[] edge
float32 threshold
float32[] center
float32 offset_angle